src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritItemListenerTest.java

```diff
@@ -92,4 +96,28 @@ public void testOnLoadedWithNoConnection() throws Exception {
         listener.onLoaded();
         Mockito.verify(gerritServer, Mockito.times(0)).startConnection();
     }
+
+    @Test
+    public void testOnJobRenamed() throws Exception {
+        FreeStyleProject job = j.createFreeStyleProject("MyJob");
+        PatchsetCreated event = Setup.createManualPatchsetCreated();
+
+        GerritTrigger trigger = spy(new GerritTrigger(
+                null, null, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                true, true, true, false, false, "", "", "", "", "", "", "", null, null, null,
+                null, false, false, "", null));
+
+        doReturn(new GerritTrigger.DescriptorImpl()).when(trigger, "getDescriptor");
+        job.addTrigger(trigger);
+        trigger.start(job, true);
+
+        when(trigger.isInteresting(event)).thenReturn(true);
+        job.renameTo("MyJobRenamed");
+        GerritItemListener listener = new GerritItemListener();
+        listener.onLocationChanged(job, "MyJob", "MyJobRenamed");
+        PluginImpl.getInstance().getHandler().notifyListeners(event);
+        Thread.sleep(100);
```

Comment:
is there a better way to give the job some time to build ? 
Just need to check it has been scheduled.